src/c++/vdo/tests/OldRecoveryJournal_t1.c

```diff
@@ -178,13 +214,14 @@ static void testDirty(void)
 {
   parameters.backingFile = crashedPath;
   initializeTest(&parameters);
-  setStartStopExpectation(VDO_READ_ONLY);
-  startVDO(VDO_DIRTY);
+  readVDOFromDisk(crashedPath);
+
+  // VDO_UNSUPPORTED_VERSION is translated to -EINVAL upon load failure
+  startVDOExpectError(-EINVAL);
+
+  checkRAMLayerContents(getSynchronousLayer(), expectedData, mismatchChecker);
   stopVDO();
```

Comment:
You should stop VDO, then check the contents. Stopping VDO could write out the superblock or make other modifications.

Reply:
Fixed.